### PR TITLE
Pt-br translation of Refuge

### PR DESCRIPTION
**Pull request by botoserelepe**
Hi @lh1008. here is the translation of "refuge" text for review. :)

### CODE REVIEW

**Comment by erciccione** [comment]
Thanks @botoserelepe 

maybe @vp11 or @netrik182 could review this?

File: monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Sob a "Excessive Fines Clause" ou Cláusula de Multas Excessivas (aplicável nos governos estaduais e locais sob a "Due Process Clause" ou Cláusula do Processo Justo) da [Oitava Emenda Constitucional dos Estados Unidos](https://constitutioncenter.org/interactive-constitution/amendments/amendment-viii), é proibida a imposição de fianças e punições cruéis, incomuns e excessivas por parte dos governos federal, estaduais e locais. Entretanto, como visto nos exemplos acima, a proibição codificada de multas excessivas é insuficiente no cenário jurídico atual.
+
+As moedas digitais não rastreáveis, como o Monero, oferecem uma privacidade sem precedentes para aqueles que guardam quantias consideráveis de dinheiro, as quais podem ser a qualquer momento tomadas por meio de multas excessivas e nunca reembolsadas. A adoção e o uso em massa de criptomoedas preservadoras de privacidade também pode levar à redução de brigas judiciais exorbitantes para recuperação de propriedades (uma vez que não há nenhum rastreio de propriedade para se ir atrás). Nessas brigas, os "politicamente fracos" e os pobres geralmente não têm voz para contestar.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md]:
```suggestion
As moedas digitais não-rastreáveis, como o Monero, oferecem uma privacidade sem precedentes para aqueles que guardam quantias consideráveis de dinheiro, as quais podem ser a qualquer momento tomadas por meio de multas excessivas e nunca reembolsadas. A adoção e o uso em massa de criptomoedas preservadoras de privacidade também pode levar à redução de brigas judiciais exorbitantes para recuperação de propriedades (uma vez que não há nenhum rastreio de propriedade para se ir atrás). Nessas brigas, os "politicamente fracos" e os pobres geralmente não têm voz para contestar.
```

File: monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+O Monero se estabelece como um pedestal de equidade nos tribunais, caso o indivíduo se encontre em uma briga judicial pelos seus direitos.
+
+
+## Últimas considerações
+
+Apesar de as criptomoedas se utilizarem de maneiras nunca antes vistas para proteger um indivíduo de abusos do governo, é importante entender que nem todas as moedas sustentam os mesmos níveis de funcionalidade e proteção.
+
+O Monero representa de fato uma moeda voltada para a privacidade, se destacando sobre as demais, como o Bitcoin. A essência do Monero foca na privacidade e na fungibilidade, e fornece uma camada extra de proteção para os indivíduos que eventualmente tiverem seu computador confiscado ou vigiado pelo governo. Adicionalmente, o Monero oferece uma plataforma para que os seus usuários guardem seus fundos de forma segura, evitando que eles se tornem vítimas de multas excessivas.
+
+Como observado acima, a mudança não é imediatamente efetiva e toma tempo até que todo o caminho até a liberdade seja traçado, passo por passo. Como os abusos de confisco não vão cessar da noite pro dia, o Monero e outras moedas voltadas à privacidade oferecem refúgio aos indivíduos controlados.
```
Comment (netrik182) [inline monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md]:
```suggestion
Como observado acima, a mudança não é imediatamente efetiva e leva tempo até que todo o caminho para a liberdade seja traçado, passo por passo. Como os abusos de confisco não vão cessar da noite para o dia, o Monero e outras moedas voltadas à privacidade oferecem refúgio aos indivíduos controlados.
```

**Comment by netrik182** [comment]
Just one last thing I didn't noticed before. The title must read `# Monero: Um refúgio...` instead of `#Monero um refúgio...` otherwise the markdown thing won't work properly.

File: monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md [hunk 3]
```diff
@@ -1,35 +1,35 @@
-#Monero: Um refúgio das severas leis de confisco civil (Parte 1)
+#Monero: Um refúgio para as severas leis de confisco civil (Parte 1)
```
Comment (lh1008) [inline monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md]: # Monero: Um refúgio para as severas leis de confisco civil (Parte 1)

Just missing a space after "#".

File: monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
+# Monero: Um refúgio para as severas leis de confisco civil (Parte 1)
+*15/04/19*
```
Comment (erciccione) [inline monero-outreach-docs/translations/pt-br/storyboard/refuge_pt-br.md]: Please add 2 trailing spaces at the end of this string to keep the formatting.

**Pull request merged**